py3status/modules/mpd_status.py

```diff
@@ -6,6 +6,10 @@
     cache_timeout: how often we refresh this module in seconds (default 2)
     format: template string (see below)
         (default '%state% [[[%artist%] - %title%]|[%file%]]')
+    format_mpd_not_authenticating: string to print
+        (default 'mpd_status: Authentication failed')
+    format_mpd_not_running: string to print
+        (default 'mpd_status: Connection refused')
```

Comment:
_(Something I was going to comment earlier in other thread... but I caved... Whatever, I comment now to get this out of the bag.)_

I like the feeling of making a string customizable by start typing off with `format_`. Even for icons too. If we change `format_clone` to `icon_clone`, then why not `separator_clone` or `symbol_clone`? 

If it does not start off with `format_`, then everybody could assume it's an option. To assign customizable strings to `error_`, `icon_` things non-format more specific seems like a hassle.

`button_`, `key_` seems okay for mouse/keyboard since there is nothing to see.

I see the benefits of using `icon_`... is that it's not a place for placeholders.
`format` keeps things simple / non-specific for everybody (imo).

----

_My response._

Maybe not `error_`... but something more generic to represent error, notification, unavailable, broken/down service, etc.

`alert_`? Idk. We'll look it up on Internet / figure out what word to use. Discuss with @Ultrabug to decide on the final word before we merge this PR as he too have bunch of `format_` in his `xrandr` script.

Last guess... `notification_` ? Will separate PRs for `max_width`.